vcd/resource_vcd_vapp_network_test.go

```diff
@@ -433,6 +632,46 @@ resource "vcd_vapp_network" "{{.resourceName}}" {
 }
 `
 
+const testAccCheckVappNetwork_update_ipv6 = `
```

Comment:
Could you also add a `vcd_vapp_vm` that uses this IPv6 network to the test case? I don't think we have any VM using IPv6 because we haven't yet supported it, but we want to be sure there is no surprise in the VM itself while using this network with IPv6 address.

The cheapest one to add in terms of testing performance is "empty vm" (the one without vApp template so it saves time on cloning the image) https://registry.terraform.io/providers/vmware/vcd/latest/docs/resources/vapp_vm#example-usage-empty-vm

Reply:
Added.